fedbiomed/common/optimizers/declearn/declearn_optimizer.py

```diff
@@ -0,0 +1,55 @@
+# This file is originally part of Fed-BioMed
```

Comment:
Hi Yannick, 

I didn't understand exactly why you preferred to import everything in __init__.py along with  creating another modules as `declearn_optimizer.py` and `declearn_regularizer`. I think it can be simple if you import everything in `fedbiomed/common/optimizers/declearn.py`. I think having 3 different files  in declearn folder is  unnecessary.  You can add `list_optim_modules` and `list_optim_regularizers` method in `declearn.py` module. There will be less files. What do you think?